cps/tasks/download.py

```diff
@@ -6,9 +6,9 @@
 from datetime import datetime
 from flask_babel import lazy_gettext as N_, gettext as _
 
-from cps.constants import XKLB_DB_FILE
-from cps.services.worker import CalibreTask, STAT_FINISH_SUCCESS, STAT_FAIL, STAT_STARTED, STAT_WAITING
-from cps.subproc_wrapper import process_open
+from ..constants import XKLB_DB_FILE
+from ..services.worker import CalibreTask, STAT_FINISH_SUCCESS, STAT_FAIL, STAT_STARTED, STAT_WAITING
+from ..subproc_wrapper import process_open
```

Comment:
Absolute imports are usually preferred because relative imports (..) can lead to confusion in module resolution.

Does an absolute path starting from the package name work?

```suggestion
from calibreweb.cps.constants import XKLB_DB_FILE
from calibreweb.cps.services.worker import CalibreTask, STAT_FINISH_SUCCESS, STAT_FAIL, STAT_STARTED, STAT_WAITING
from calibreweb.cps.subproc_wrapper import process_open
```

Reply:
Absolute paths work. I was not comfortable seeing them flagged as unresolved in vscode. Thanks.